.gitignore

```diff
@@ -1 +1,2 @@
-docs/_generated
\ No newline at end of file
+docs/_generated
+tmp*
```

Comment:
I was using this extra `tmp*` to put my test files in the root of the project with names such as `tmp.typ` and `tmp.png`. I hope it's okay to add this one to the `.gitignore`.